book/03-git-branching/sections/remote-branches.asc

```diff
@@ -6,13 +6,17 @@ Remote references are references (pointers) in your remote repositories, includi
 You can get a full list of remote references explicitly with `git ls-remote [remote]`, or `git remote show [remote]` for remote branches as well as more information.
 Nevertheless, a more common way is to take advantage of remote-tracking branches.
 
-Remote-tracking branches are references to the state of remote branches.
-They're local references that you can't move; they're moved automatically for you whenever you do any network communication.
-Remote-tracking branches act as bookmarks to remind you where the branches in your remote repositories were the last time you connected to them.
-
-They take the form `(remote)/(branch)`.
-For instance, if you wanted to see what the `master` branch on your `origin` remote looked like as of the last time you communicated with it, you would check the `origin/master` branch.
-If you were working on an issue with a partner and they pushed up an `iss53` branch, you might have your own local `iss53` branch; but the branch on the server would point to the commit at `origin/iss53`.
+Remote-tracking branches are references to the state of remote branches.  They're
+local references that you can't move; they're moved automatically for you whenever
+you do any network communication, and they act as bookmarks to remind you where the
+branches in your remote repositories were the last time you connected to them.
+
+Remote-tracking branches take the form `<remote>/<branch>`.  For instance, if you
+wanted to see what the `master` branch on your `origin` remote looked like as of
+the last time you communicated with it, you would check the `origin/master` branch.
+If you were working on an issue with a partner and they pushed up an `iss53` branch,
+you might have your own local `iss53` branch, but the branch on the server would
+be represented by the commit on your remote-tracking branch `origin/iss53`.
```

Comment:
Doubled spaces; I see you're a fan of Emacs' `M-q`. 😉 Line breaks, please; I had to switch to the rendered diff to figure out what you had changed.

The first change in this block is to remove a semicolon and join the following sentence on. I was trying to juxtapose "you can't move this branch" and "Git moves it for you," but adding the third clause removes the focus on that fact. Maybe something like this:

> They’re local references that you can’t move; Git moves them for you whenever you do any network communication, to make sure they accurately represent the state of the remote repository. Think of them as bookmarks, to remind you where the branches in your remote repositories were the last time you connected to them.

There's another change that isn't quite right:

> […] the branch on the server would be represented by the commit on your remote-tracking branch `origin/iss53`.

Nope. Branches aren't represented by commits. I know I've said that we don't want to draw super-technical distinctions this early in the book, but we also don't want to set the reader up to be confused until chapter 10, when we finally get pedantic about definitions. A better way to say this would be:

> […] the branch on the server would be represented by the remote-tracking branch `origin/iss53`.